cohort_job_server/sjs_api/cohort_requests/base_cohort_request.py

```diff
@@ -50,20 +49,21 @@ def create_sjs_request(self, cohort_query: CohortQuery) -> str:
         if cohort_query is None:
             raise FhirException("No query received to format.")
 
-        is_pseudo = self.is_cohort_request_pseudo_read(username=self.__headers_to_owner_entity(),
+        is_pseudo = self.is_cohort_request_pseudo_read(username=self.owner_username,
                                                        source_population=cohort_query.source_population.care_site_cohort_list)
 
         sjs_request = QueryFormatter(self.auth_headers).format_to_fhir(cohort_query, is_pseudo)
         cohort_query.criteria = sjs_request
 
         callback_path = self.callback_path or (
                 self.mode == Mode.COUNT_WITH_DETAILS and f"/cohort/feasibility-studies/{cohort_query.instance_id}/" or None)
-        spark_job_request = SparkJobObject(cohort_definition_name="Created from Django",
+        spark_job_request = SparkJobObject(cohort_definition_name="Created from C360 backend",
                                            cohort_definition_syntax=cohort_query,
                                            mode=self.mode,
-                                           owner_entity_id=self.__headers_to_owner_entity(),
+                                           owner_entity_id=self.owner_username,
                                            callbackPath=callback_path,
-                                           existingCohortId=self.existing_cohort_id
+                                           existingCohortId=self.existing_cohort_id,
+                                           samplingRatio=self.sampling_ratio
```

Comment:
the param will be `modeOptions={"sampling": self.sampling_ratio}` instead of `samplingRatio`

Reply:
you also have to modify `format_spark_job_request_for_sjs` accordingly

Reply:
also for the `format_spark_job_request_for_sjs` it uses [HOCON format](https://github.com/lightbend/config?tab=readme-ov-file#using-hocon-the-json-superset) so you can mix `xxx = yyy` with normal json `xxx = {"a": "b"}`.

and the sampling ratio is sent as a string